src/views/Login.vue

```diff
@@ -8,23 +8,47 @@
     <p>
       Når du sitter i hjemmekarantene skal du ikke oppholde deg på steder der
       man lett kommer nær andre. Da kan det bli veldig vanskelig å få handlet
-      inn.
+      inn det man trenger til livets opphold.
     </p>
     <p>
-      Karantenehjelpen skal hjelpe de som sitter i karantene å komme i kontakt
-      med friske folk som ønsker å hjelpe.
+      Karantenehjelpen er til for å hjelpe de som sitter i karantene å komme i
+      kontakt med friske folk som ønsker å hjelpe.
     </p>
     <p>
-      Vi trenger både folk som er friske og vil stille opp for sine naboer, og
-      tilbyr handlehjelp til de som trenger det mest.
+      Vi trenger friske folk som har lyst til å delta i den nasjonale dugnaden
+      ved å hjelpe dem som sitter i karantene med innkjøp.
```

Comment:
Vi gjør mer enn innkjøp nå som vi har åpnet for andre henvendelser. Så dette burde være mer generelt .

Reply:
Jeg oppdaterte teksten litt her https://github.com/s0lvang/Karantenehjelpen/blob/project/setup/README.md så du kan sikkert koke litt av det :)

Reply:
Fosskokte bare teksten fra README, den var fin